src/wp-admin/includes/upgrade.php

```diff
@@ -878,6 +878,10 @@ function upgrade_all() {
 		upgrade_560();
 	}
 
+	if ( $wp_current_db_version < 51769 ) {
```

Comment:
I chose this number arbitrarily, I guess we should discuss it more before merging.